src/ethereum/info/zksyncInfo.ts

```diff
@@ -96,11 +96,7 @@ const networkInfo: EthereumNetworkInfo = {
     },
     {
       type: 'evmscan',
-      servers: [
-        'https://block-explorer-api.mainnet.zksync.io',
-        'https://api-era.zksync.network',
-        'https://zksync.blockscout.com/api'
```

Comment:
Is this url broken? We add /api in fetchGetEtherscan

Reply:
The url is broken regardless of if /api is added, omitted this url.